packages/core/graph/src/shared-buffer.js

```diff
@@ -1,5 +1,4 @@
 // @flow
-/* global MessageChannel:readonly */
```

Comment:
This is likely due to the update to eslint

![Screenshot from 2023-01-11 15-23-12](https://user-images.githubusercontent.com/12656294/211717229-151ff4fe-886a-4754-ad47-2120e659a6dd.png)